internal/runtime/compiler/parser/parser.y

```diff
@@ -112,7 +117,9 @@ stmt_list
 
 /* Types of statements. */
 stmt
-  : conditional_stmt
+  : log_mapping_spec %prec LOGMAPPING
```

Comment:
For consistency make this a `log_filter_stmt` and then create the intermediate `log_fitler_stmt` where you statr the log_mapping_spec on line 152.